asv_bench/benchmarks/io/csv.py

```diff
@@ -76,6 +76,41 @@ def time_frame(self, obs):
         self.data.to_csv(self.fname)
 
 
+class ToCSVIndexes(BaseIO):
+
+    fname = "__test__.csv"
+
+    def setup(self):
+        ROWS = 1000000
+        COLS = 5
+        N_HEAD = 100000
+        index_cols = {
+            "index1": np.random.randint(0, ROWS, ROWS),
+            "index2": np.full(ROWS, 1, dtype=np.int),
+            "index3": np.full(ROWS, 1, dtype=np.int),
+        }
+        data_cols = {
+            f"col{i}": np.random.uniform(0, 100000.0, ROWS) for i in range(COLS)
+        }
+        df = DataFrame({**index_cols, **data_cols})
+        self.df_standard_index = df.head(N_HEAD)
```

Comment:
the HEAD is a bit confusing here. can you not just make ROWS=N_HEAD to achieve the same?

Reply:
It's like that because the following two tests do take a `head` with a smaller number of rows, and I didn't want to duplicate the creation code. However, that's easily solved by extracting that into a function.

Reply:
oh it was fine, but ok on the change. ping when green.